api/base/settings/defaults.py

```diff
@@ -51,21 +51,24 @@
 }
 
 REST_FRAMEWORK = {
-    'PAGE_SIZE': 10,
-
-    # Order is important here because of a bug in rest_framework_swagger. For now,
```

Comment:
Ah, that's the problem, then. Add `'rest_framework.renderers.JSONRenderer', ` back to the `DEFAULT_RENDERER_CLASSES` at the top and you should have swagger docs again.